### PR TITLE
Create layout logic for editing profile

### PR DESCRIPTION
**Pull request by naberalgamma**

### CODE REVIEW

**Review by 6mahdapihka9 — CHANGES_REQUESTED** [review]
check review in Vasyl's PR

File: frontend/src/components/Profile/BasicTabs.tsx [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
-import { Tabs, Tab, Typography, Box } from '@mui/material';
+// import { useState } from 'react';
+import { useTypedSelector } from 'redux/hooks/useTypedSelector';
+import { Tabs, Tab, Typography, Box, TextField } from '@mui/material';
+import { Controller, Control } from 'react-hook-form';
+import { UserForm } from './types';
 
 interface TabPanelProps {
-  children?: React.ReactNode;
+  children?: any;
```
Comment (Yojuks) [inline frontend/src/components/Profile/BasicTabs.tsx]: React.ReactNode - why this code doesn`t work?

File: frontend/src/components/Profile/ProfilePage.tsx [hunk 2]
```diff
@@ -1,58 +1,202 @@
...
+  email,
+  displayName,
+  createdAt
+}: ProfilePageProps) {
+  const dispatch = useDispatch();
+  const { handleSubmit, control, register } = useForm<UserForm>({
+    mode: 'onBlur',
+    defaultValues: {
+      displayName
+      // description
```
Comment (Yojuks) [inline frontend/src/components/Profile/ProfilePage.tsx]: delete

File: frontend/src/components/Profile/styles.ts [hunk 3]
```diff
@@ -1,11 +1,11 @@
-import { Box, Avatar } from '@mui/material';
+import { Box, Avatar, TextField } from '@mui/material';
 import { styled } from '@mui/material/styles';
 import Button from '@mui/material/Button';
 // import { Typography } from '@mui/material';
```
Comment (Yojuks) [inline frontend/src/components/Profile/styles.ts]: delete

**Review by Yojuks — CHANGES_REQUESTED** [review]

**Review by Yojuks — APPROVED** [review]

**Review by 6mahdapihka9 — APPROVED** [review]
👌

**Pull request merged**